kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/CreateQuizSection.vue

```diff
@@ -491,19 +296,25 @@
 <style lang="scss"  scoped>
 
   .style-icon {
-    width: 2.5em;
-    height: 2.5em;
```

Comment:
What could be the reason for not maintaining em units ? as we discussed.

Reply:
Old habits die hard 😅 - will update to use em